lang/en.json

```diff
@@ -463,6 +463,8 @@
         "SmtpServerAddress": "SMTP server address",
         "SmtpUsername": "SMTP username",
         "Source": "Source",
+        "Star": "Star",
+        "StarredBy": "Starred by",
```

Comment:
I think using `starred` and `starred_by` in the code is fine, but in the UI using star as a verb is non-standard. We should probably rephrase it to something like "Save as favourite" or similar.